schemas/BaseElements.xsd

```diff
@@ -2709,6 +2709,11 @@ http://www.epa.gov/compliance/monitoring/programs/caa/woodheaters.html</xs:docum
 			<xs:extension base="HVACSystemInfo">
 				<xs:sequence minOccurs="0">
 					<xs:element name="HeatPumpType" type="HeatPumpType" minOccurs="0"/>
+					<xs:element minOccurs="0" name="ColdClimate" type="xs:boolean">
+						<xs:annotation>
+							<xs:documentation>Is this a cold climate heat pump?</xs:documentation>
```

Comment:
@nmerket @gmdickey Glenn and I have discussed this before - it is problematic as there is no official designation, just performance curves that are better at lower temperature.